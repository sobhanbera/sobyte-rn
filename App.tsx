/**
 * Â© Sobyte
 *
 * @author : Sobhan Bera (sobhanbera)
 * @other_editors :
 * @file : TyepscriptReact
 *
 * Purpose - application's entry point wrapper after "index.js"
 */

import React from 'react'
import 'react-native-gesture-handler'

import {AppEntryPoint} from './app/export'

const App = () => <AppEntryPoint />

export default App
