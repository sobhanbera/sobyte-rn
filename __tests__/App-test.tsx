/**
 * @format
 */

import 'react-native'
import React from 'react'
import App from '../App'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

const response: any = {} // just a bare variable to reduce error in resolve statement below...
global.fetch = jest.fn(
    () => new Promise((resolve, _reject) => resolve(response)),
)
jest.mock('react-native-gesture-handler', () => {})

it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree).toMatchSnapshot()
})
