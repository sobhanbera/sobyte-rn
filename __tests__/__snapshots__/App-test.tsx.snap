// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <Text>
    Sobyte - Entry Point
  </Text>
</RCTSafeAreaView>
`;
